RabbitMQ, GoogleTest, SimpleAmpqClient/GoogleTest.md

```diff
@@ -0,0 +1,30 @@
+#GoogleTest.md
+
+##GoogleTest 
+
+Testing the telemetry test tool and Hermes is done with googletest and google mock 
```

Comment:
Instead of being specific on which programs use Google Test, we can provide a general overview of what google test is and how it's a C++ testing framework.